source/cellular_3gpp_api.c

```diff
@@ -26,7 +26,7 @@
  */
 
 /**
- * @brief FreeRTOS Cellular Library API implemenation with 3GPP AT command.
+ * @brief FreeRTOS Cellular Library API [implementation with 3GPP AT command.
```

Comment:
Is the open bracket here for a reason?

Reply:
Was an accident, thanks for catching it, removed it.